accounts/abi/unpack_pack_fuzz_test.go

```diff
@@ -0,0 +1,320 @@
+// (c) 2019-2023, Ava Labs, Inc. All rights reserved.
+// See the file LICENSE for licensing terms.
+
+//go:build fuzz
```

Comment:
There was a g++/cppgc link problem with v8go package in test runs: https://github.com/ava-labs/subnet-evm/actions/runs/6866130434/job/18671676772?pr=993.

I did not want to change our whole build flags just because of this tests, as these will run every day with `fuzz.yml` CI flow. So I just ignored this in build in tests and added specific `fuzz` tag.